install/kubernetes/Makefile

```diff
@@ -22,6 +22,8 @@ LATEST_VERSION_REGEX := '[0-9]\+\.[0-9]\+\.90'
 DEV_VERSION_REGEX := '[0-9]\+\.[0-9]\+-dev'
 CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_PULLPOLICY_REGEX := '\([pP]ullPolicy:\) .*'
+QUICK_OPTIONS := \
+    --set hubble.tls.enabled=false
```

Comment:
Reviewer note: This may look a bit dangerous, but with `hubble.listenAddress` also being nil, we don't serve the Hubble API via TCP, so TLS being disabled in quick-install does expose anything.

Reply:
I assume that by:

> we don't serve the Hubble API via TCP, so TLS being disabled in quick-install does expose anything.

you mean:

> we don't serve the Hubble API via TCP, so TLS being disabled in quick-install doesn't expose anything.